docker/Dockerfile.base

```diff
@@ -7,7 +7,7 @@ RUN apk add --no-cache libc6-compat
 # Install pnpm
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
-RUN corepack enable && corepack prepare pnpm@8.6.3 --activate
+RUN corepack enable && corepack prepare pnpm@8.15.6 --activate
```

Comment:
And improvement for the future is pass this as build argument. To have more centralized